debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/transforms/partitions/ComputePartitionTest.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.mysql.transforms.partitions;
+
+import static io.debezium.data.Envelope.Operation.CREATE;
+import static io.debezium.data.Envelope.Operation.DELETE;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+
+import io.debezium.data.Envelope;
+
+public class ComputePartitionTest {
+
+    public static final Schema VALUE_SCHEMA = SchemaBuilder.struct()
+            .name("mysql-server-1.inventory.product.Value")
+            .field("id", Schema.INT64_SCHEMA)
+            .field("price", Schema.FLOAT32_SCHEMA)
+            .field("product", Schema.STRING_SCHEMA)
+            .build();
+    public static final Schema SOURCE_SCHEMA = SchemaBuilder.struct()
+            .name("source")
+            .field("table", Schema.STRING_SCHEMA).build();
+
+    private final ComputePartition<SourceRecord> computePartitionTransformation = new ComputePartition<>();
+
+    @Test
+    public void correctComputeKafkaPartitionBasedOnConfiguredFieldOnCreateAndUpdateEvents() {
+
+        configureTransformation("products", "products:product", "products:2");
```

Comment:
PLease use fully-qualified table names in examples. It should have no visible impact but let's be sure.